package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@medibloc/panacea-js",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
Please revert this version number for now. After merging this PR, I will create a release commit.

Reply:
Okay. I fixed it.